src/panels/config/logs/error-log-card.ts

```diff
@@ -268,6 +311,10 @@ class ErrorLogCard extends LitElement {
         11
       );
     }
+    if (this._downloadSupported === undefined && this.hass) {
+      this._downloadSupported =
+        !isIosApp(this.hass) || downloadFileSupportedViaApp(this.hass);
```

Comment:
The `!isIosApp(this.hass)` condition can be moved inside `downloadFileSupportedViaApp` function.